source/LocalizationManager.cs

```diff
@@ -2,28 +2,36 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Globalization;
+using System.Linq;
 
 namespace VersionOne.Localization
 {
 	public class LocalizationManager
 	{
 		private readonly CultureInfo _defaultculture;
-		private readonly ITemplateSetLoader _loader;
-		private readonly string[] _setnames;
+		private readonly IEnumerable<ITemplateProvider> _providers; 
 		private readonly IDictionary _localizers;
 		private readonly IDictionary<string, string> _overrides;
 
-		public LocalizationManager(string defaultculture, ITemplateSetLoader loader, IDictionary<string, string> overrides, params string[] setnames)
-			: this(new CultureInfo(defaultculture), loader, setnames)
+		public LocalizationManager(string defaultculture, ITemplateSetLoader loader, IDictionary<string, string> overrides,
+			params string[] setnames)
+			: this(
```

Comment:
Could this simply delegate to the constructor below to DRY up provider set creation?

`: this(new CultureInfo(defaultculture), loader, setnames)`